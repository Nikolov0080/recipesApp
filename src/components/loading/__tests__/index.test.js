import React from 'react'
import ReactDOM from 'react-dom';
import Loading from '../index';
import {  cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders Loading OK', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Loading />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot', () => {
    const three = renderer.create(<Loading/>).toJSON();
    expect(three).toMatchSnapshot()
});