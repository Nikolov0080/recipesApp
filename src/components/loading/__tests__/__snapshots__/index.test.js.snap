// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="loading_container"
>
  <span
    className="sc-AxjAm jfcyvh"
    data-testid="skeleton__item"
    height="1000px"
    width="100%"
  >
    <h1
      className="pulsate"
    >
      Loading...
    </h1>
  </span>
</div>
`;
